cdbot/__init__.py

```diff
@@ -10,5 +11,5 @@
 def main():
     """Entry point for poetry script."""
     if environ.get("SENTRY_URL") is not None:
-        sentry_sdk.init(environ.get("SENTRY_URL"))
-    bot.run(environ.get("BOT_TOKEN"))
+        sentry_sdk.init(base64.b64decode(environ.get("SENTRY_URL")))
```

Comment:
This is not nice, could this be stored in the constants file so you can pass it straight into the function?

Reply:
I've not moved this to constants.py as it is only called once.